### PR TITLE
SAK-47945 Assignments make resubmission date match live date of close date

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

File: assignment/tool/src/webapp/js/assignments.js [hunk 1]
```diff
@@ -969,6 +969,26 @@ ASN.changeVisibleDate = function()
...
+    ASN.resizeFrame();
+  }
+  else
+  {
+    document.getElementById('allowResubmitNumber').style.display = 'none';
+    document.getElementById('allowResubmitTime').style.display = 'none';
+    document.getElementById('resubmitNotification').style.display = 'none';
+  }
+}
+
```
Comment (kunaljaykam) [inline assignment/tool/src/webapp/js/assignments.js:991]:
```suggestion
ASN.changeResubmissionDate = function() {

  const displayStyle = document.getElementById("allowResToggle").checked ? 'block' : 'none';

  const elementIds = ['allowResubmitNumber', 'allowResubmitTime', 'resubmitNotification'];

  elementIds.forEach(id => document.getElementById(id).style.display = displayStyle);

  if (displayStyle === 'block') {
  
    document.getElementById('resubmissiondate').isoDate = document.getElementById('closedate').isoDate;
  }
};

```

**Review by kunaljaykam — APPROVED** [review]
Everything looks good to me. I have left a suggestion for the js function to be more concise.

File: assignment/tool/src/webapp/js/assignments.js (hunk 1 above)
Comment (ottenhoff) [reply assignment/tool/src/webapp/js/assignments.js:991]: i smell chatgpt ;)

**Comment by kunaljaykam** [comment]
It's adrianGPT. He loves one line JS. I love it too now.

File: assignment/tool/src/webapp/js/assignments.js (hunk 1 above)
Comment (ottenhoff) [reply assignment/tool/src/webapp/js/assignments.js:991]: ultimately, the way this is done with three separate DIVs being hidden/shown is bad. I considered consolidating it down to one container DIV but wasn't ready to spend that much time on this one.

**Pull request merged**